src/js/liblab/LabObjects.js

```diff
@@ -886,7 +886,7 @@ labObjects.Environment = {
     proceedMotion: function (beh, dt, i, a) {},
 
     calculateForces: function (beh) {
-        var i, m1, x1, y1, j, m2, x2, y2, k, fx, fy, r, l;
+        let i, m1, x1, y1, j, m2, x2, y2, k, fx, fy, r, l;
```

Comment:
Remove `i` and `j` from `let` statement here and move to the `for` statements (7 places). In fact it appears all of these variables with the exception of `fx` and `fy` in one place where they assigned to `0` could be declared `const` in each block where they are assigned below as they are only handles and are not changed later in their code blocks. The place where `fx = 0;fy = 0;` occurs then becomes `let fx, fy;` as the decision tree always assigns a value to them so there is no need to set them to `0`.